conf/messages

```diff
@@ -212,6 +212,11 @@ annotation.makeHybrid.failed=Could not convert to hybrid.
 annotation.makeHybrid.alreadyHybrid=Could not convert annotation to hybrid annotation because it is already a hybrid annotation.
 annotationType.notFound=Could not find the specified annotation type.
 annotation.cantMergeVolumes=This annotation collection contains volume tracings. Merging volume tracings is not yet supported. Please use download instead.
+annotation.undoFinish.success=Reactivated Annotation
+annotation.undoFinish.tooLate=It's not possible to reactivate an annotation if it was finished more than 30 seconds ago.
```

Comment:
Can you import the 30 seconds from the config? If not, I'd simply rephrase it so that the time is not explicitly mentioned (e.g., "the annotation cannot be reactivated anymore, since it's been closed for too long.")